src/test/java/Homework18_PB.java

```diff
@@ -1,3 +1,9 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
 public class Homework18_PB extends BaseTest{
```

Comment:
Please, avoid own names or vague words: If there’s not a real need for them, avoid using these words, which often add no value

For exampe

`public class HomeWork18 `